### ISSUE TITLE
Listbox-Button: allow ability to construct label from external element plus button value

### ISSUE THREAD

**Issue by ianmcburnie** [eBay/ebayui-core#1321]
# Feature Request

## Description

Allow ability to construct label from external text element plus button value

Example:

```html
<span id="el1">Reason</span>
<button aria-labelledby="el1 el2"><span id="el2">Cannot contact seller</span></button>
```

## Context

We often see teams trying to use listbox-button like a `<select>`, by providing a `<label>` tag. However, a `<label>` tag does not work for buttons. We would need to allow the accessible label to be constructed via a space separated list of ids using `aria-labelledby`.

## Screenshot

![Screen Shot 2021-02-03 at 10 44 03 AM](https://user-images.githubusercontent.com/38065/106793843-d4288c80-660c-11eb-86e6-0431f55dd3d8.png)

**Comment by agliga** [comment]
This has been published in `6.1.0` and in 7.2.0` beta for now (will go out in `7.2.0`)
